app/screens/sso/sso_with_redirect_url.tsx

```diff
@@ -57,12 +57,23 @@ function SSOWithRedirectURL({
         });
         const url = parsedUrl.toString();
 
-        const onError = () => setError(
-            intl.formatMessage({
-                id: 'mobile.oauth.failed_to_open_link',
-                defaultMessage: 'The link failed to open. Please try again.',
-            }),
-        );
+        const onError = (e: Error) => {
+            let message;
+            if (e?.message?.match(/no activity found to handle intent/i)) {
```

Comment:
This applies to Android only, right? What if the device is not in english? Does the message match?

Reply:
Yes the message is system generated and is matching different language ones too  ... Just now checked with Spanish.